workspace/eval_audio_gpt_lora.py

```diff
@@ -0,0 +1,226 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
what is this workspace folder?

Reply:
some example tools for picking off experiments.

Reply:
Add a TODO since undecided yet. leave this to the later.